pkg/storage/chunk/storage/caching_index_client.go

```diff
@@ -5,9 +5,11 @@ import (
 	"sync"
 	"time"
 
+	util_log "github.com/cortexproject/cortex/pkg/util/log"
```

Comment:
We have forked the `log` package from Cortex which is what we are using now. Please use `util_log "github.com/grafana/loki/pkg/util/log"`.

Reply:
done，thanks